orchagent/zmqorch.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <vector>
+#include <string>
+#include <orch.h>
+#include "zmqserver.h"
+
+class ZmqConsumer : public ConsumerBase {
```

Comment:
This does not have all the consumer methods. How does warm boot work?
Looks like the current codes just cast the executor to be Consumer in the warmboot logic. It won't work for ZmqConsumer.

Reply:
I confirmed that currently the Dash no plan for warm-reboot support.

Reply:
I have two PRs:
https://github.com/sonic-net/sonic-swss/pull/2857
https://github.com/sonic-net/sonic-swss-common/pull/803

Could you take a look at them please?